motherLang/out/production/motherLang/MotherLang.g4

```diff
@@ -308,12 +373,3 @@ NUMBER	: '-'? [0-9]+ ('.' [0-9]+)?
 		;
 
 WS	: (' ' | '\t' | '\n' | '\r') -> skip;
-
-AND : '&&'
-    ;
-
-OR  : '||'
-    ;
-
-OPBINARY  : AND | OR
```

Comment:
AND, OR e BINARY nao eram utilizados, em PRs futuros tentarei add novamente mais condiçoes no comandoSelecao 'IF'